repository-management.md

```diff
@@ -0,0 +1,101 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+Package names for WP-CLI commands must end with the `-command` suffix.
```

Comment:
... for clarity, and also to distinguish them from the non-command repositories.

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/51d8b06cf4c4953a2a1e5d2b4bf609a67818ddc0